openLRSng/TX.h

```diff
@@ -267,7 +542,7 @@ void bindMode(void)
   bool  sendBinds = 1;
 
   tx_buf[0] = 'b';
-  memcpy(tx_buf + 1, &bind_data, sizeof(bind_data));
+  memcpy((tx_buf + 1), &bind_data, sizeof(bind_data));
```

Comment:
Why the unnecessary brackets? In my opinion, brackets should only be used if they are needed, because this will result in consistent style that is not affected by judgement / preference.

Reply:
I guess this is more of a preference, although I have seen it as somewhat of a 'standard' in other cases / other code. The rationale is that it makes it clearer that the enclosed statement in brackets is being passed to the function as a clearly distinct "unit" as compared to the other arguments.

In this case, it's very obvious and straight-forward, but for more complex function calls, with more  arguments and/or other statements as arguments, it helps to keep things nice & tidy.

My vote is for the extra brackets, but you'd obviously have the final say on this.

Reply:
One of the things I am big about is consistency. And the problem with using unneeded brackets to improve clarity is that it is down to subjective judgement as to what is 'clear' and what isn't, making it hard to define rules that allow consistent use of brackets for clarity. Over time, this will lead to a lot of churn when different people edit the same file, and change brackets to whatever they perceive as the right level of clarity, making it harder overall to do reviews and keep track of actual change.

Reply:
although I don't quite agree that the extra brackets make things less clear, I guess I'll have to go along with it in this case.

I can try to clean this up if you let me know what the standard should be for brackets with expressions